pkg/crd/crds.go

```diff
@@ -57,7 +57,7 @@ const (
 )
 
 func ApplyCRDs(ctx context.Context, dynamicClient dynamic.Interface, clientset kubernetes.Interface, clientsetExt apiextensionsclientset.Interface, logger logr.Logger) error {
-	ctxLimited, cancel := context.WithTimeout(ctx, 30*time.Second)
+	ctxLimited, cancel := context.WithTimeout(ctx, time.Minute)
```

Comment:
Any special reason ?

Reply:
This is a work around for the context timeout.

If we look at line 68 of the attached cm.log in the opening commit, we can see that the 30 seconds is used up. What @pdettori did as the first-aid is to extend this timeout. So even without changes in this PR, the controller can still (slowly) starts.

We can see that now the controller starts in a few seconds from the updated log (from 16:34:38.XXX to 16:34:39.XXX). But I still keep this work around just for robustness.